code/controllers/subsystem/processing/greyscale.dm

```diff
@@ -47,6 +58,19 @@ PROCESSING_SUBSYSTEM_DEF(greyscale)
 
 	return SS_INIT_SUCCESS
 
+/datum/controller/subsystem/processing/greyscale/PostInit()
```

Comment:
why do we need to do this as a separate proc?

Reply:
this seems weird to me, I can't think of a usecase where we would want this proc that doesn't involve just dancing around guard statements like this